packages/core/src/sass/app/_common/components/onfido-container.scss

```diff
@@ -22,8 +26,15 @@
     @include mobile() {
         padding: 1.6rem;
 
-        .onfido-sdk-ui-PageTitle-titleSpan {
-            font-size: 2rem;
+        & .onfido-sdk-ui-PageTitle {
+            &-titleWrapper {
+                margin: 2.4rem 0 0;
+            }
+
+            &-title {
+                font-size: 2rem;
+                text-align: center;
```

Comment:
just a doubt, will it be possible to pass these 2 as props to `Text` component?

Reply:
same here - there are internal onfido classes